common/src/main/java/com/flipkart/varadhi/utils/HostUtils.java

```diff
@@ -15,4 +15,12 @@ public static String getHostName() throws UnknownHostException {
         log.debug("getHostName: completed");
         return host;
     }
+
+    public static String getHostAddress() throws UnknownHostException {
```

Comment:
dont log. dont throw

Reply:
faced the same issue internally. where calling this function unnecesarilly requires exception handling.

call this function at the app startup and if it throws exception fail the startup itself. If it works once, I think it will work always, hence there is no need to throw checked exception.

Reply:
Done, now at startup HostUtils init is required. Post that get calls doesn't need to handle exceptions.